### PR TITLE
Employer-profile. create entities.

### PR DESCRIPTION
**Pull request by Pchelnikova**

### CODE REVIEW

File: JobHunter.Domain/Interfaces/IEmployerProfileService.cs [hunk 1]
```diff
@@ -8,6 +8,6 @@ namespace JobHunter.Domain.Interfaces
 {
     public interface IEmployerProfileService
     {
-         bool SetEmployerProfile(EmployerProfileDomainModel model);
+        bool CreateEmployerProfile(EmployerProfileDomainModel model);
```
Comment (martyniukroman) [inline JobHunter.Domain/Interfaces/IEmployerProfileService.cs]: Look, dear. If your field is bool type by convention its recommended to use "Is" before the field name
Try like this: IsEmployerProfileCreated 
:)

File: JobHunter.Domain/Models/EmployerProfileDomainModel/EmployerProfileDomainModel.cs [hunk 2]
```diff
@@ -6,21 +6,28 @@ namespace JobHunter.Domain.Models.EmployerProfileDomainModel
...
-        public string Adress { get; set; }
-        public string Description { get; set; }
-        public string Password { get; set; }
-        public string INN { get; set; }
-        public string EGRPOU { get; set; }
+        public string companyName { get; set; }
+        public string personName { get; set; }
+        public string personSurname { get; set; }
+        public string personMiddleName { get; set; }
+        public List<string> telephone { get; set; }
```
Comment (martyniukroman) [inline JobHunter.Domain/Models/EmployerProfileDomainModel/EmployerProfileDomainModel.cs]: If you have a collection of any items use names in plural
Like this: Telephones

File: JobHunter.Domain/Interfaces/IEmployerProfileService.cs (hunk 1 above)
Comment (NerevitVasyan) [reply JobHunter.Domain/Interfaces/IEmployerProfileService.cs]: we choose our convention ourselves

Comment (Pchelnikova) [reply JobHunter.Domain/Interfaces/IEmployerProfileService.cs]: Thanks! Code convention is important and interesting thing

File: JobHunter.Domain/Models/EmployerProfileDomainModel/EmployerProfileDomainModel.cs (hunk 2 above)
Comment (Pchelnikova) [reply JobHunter.Domain/Models/EmployerProfileDomainModel/EmployerProfileDomainModel.cs]: Agree with you, thanks!

**Comment by vitalii77787** [comment]
good

**Pull request merged**